djangocms_blog/admin.py

```diff
@@ -19,6 +21,47 @@
 from .models import BlogCategory, Post
 from .settings import get_setting
 
+signal_dict = {}
+
+
+def register_extension(klass):
+    if issubclass(klass, admin.TabularInline):
```

Comment:
any reason to restrict this to `TabularInline`? Why not `InlineModelAdmin`?

Reply:
Good point.